ui/components/Connect/connections/helm-connect/StepperContent.js

```diff
@@ -0,0 +1,369 @@
+        items: {
+          enum: availableRepos,
+          type: 'string',
+        },
+        minItems: 1,
+        title: 'Select one or more helm chart',
+        type: 'array',
+        uniqueItems: true,
+        'x-rjsf-grid-area': 12,
+      },
+    },
+    required: ['selectedHelmRepos'],
+    type: 'object',
+  };
+
+  const handleChange = (data) => {
+    setFormState(data);
+  };
+
+  return (
+    <StepperContent
+      {...SelectRepositoryContent}
+      handleCallback={handleCallback}
+      disabled={!formState}
+    >
+      <RJSFWrapper
+        key="select-helm-repo-rjsf-form"
+        formData={formState}
+        onChange={handleChange}
+        jsonSchema={schema}
+        liveValidate={false}
+        formRef={formRef}
+        // setErrors={setErrors}
+      />
+    </StepperContent>
+  );
+};
+
+const HelmChartsStatus = ({ configuredRepository, handleInstall }) => {
+  const router = useRouter();
+  const theme = useTheme();
+  const meshMapRedirect = (id) => {
+    let letHyperLink = `https://playground.meshery.io/extension/meshmap?application=${id}`;
+    router.replace(letHyperLink);
+  };
+  return (
+    <Stack
+      style={{ width: '100%', marginBottom: '2rem', height: '21rem', overflow: 'auto' }}
+      spacing={2}
+    >
+      {configuredRepository?.map((item, index) => {
+        const { name, status, id } = item;
+        const isSuccess = status === 'created';
+        return (
+          <Alert
+            key={index}
+            icon={false}
+            severity={isSuccess ? 'success' : 'error'}
+            // color: isSuccess ? theme.palette.success.dark : theme.palette.error.dark
+            style={{
+              display: 'flex',
+              alignItems: 'center',
+              lineHeight: '0',
+              color: '#000',
+              backgroundColor: isSuccess ? theme.palette.keppelGreenLight : null,
+            }}
+            action={
+              !isSuccess ? (
+                <Button
+                  style={{ backgroundColor: theme.palette.error.main, color: theme.palette.white }}
+                  size="small"
+                  onClick={() => handleInstall(item)}
+                >
+                  Retry
+                </Button>
+              ) : (
+                <Button
+                  style={{ color: theme.palette.keppelGreen }}
+                  size="small"
+                  onClick={() => meshMapRedirect(id)}
+                >
+                  <DesignsIcon width="28" height="28" fill={theme.palette.keppelGreen} />
+                  <p style={{ marginLeft: '0.2rem' }}>Open Design</p>
+                </Button>
+              )
+            }
+          >
+            <AlertTitle style={{ fontSize: '1.4rem' }}>{name}</AlertTitle>
+            {isSuccess ? (
+              <div>
+                <p>{`Design ${name} created`}</p>
+              </div>
+            ) : (
+              <div>
+                <p>{`Unable to import ${name}.yaml`}</p>
```

Comment:
```suggestion
                <p>{`Unable to import <name of chart>`}</p>
```
@Yashsharma1911 this should contain the name of chart/<and the version of the chart>

Reply:
fixed